app/qml/project/MMProjectHomeTab.qml

```diff
@@ -230,29 +116,39 @@ Item {
       topMargin: root.spacing
     }
 
+    activeProjectAlwaysFirst: true
+    projectsModel.activeProjectId: root.activeProjectId
+
+    list.section {
+      property: "ProjectIsActiveProject"
+      criteria: ViewSection.FullString
+      delegate: Column {
+        width: ListView.view.width
+
+        Text {
```

Comment:
Can we use `MMText` here, please? It automatically sets the line height (you should not need to set `height` explicitly then)